### ISSUE TITLE
Fluid storage bus cannot see if a recipe is completed

### ISSUE THREAD

**Issue by mertgoksel** [GlodBlock/AE2FluidCraft-Rework#148]
I have tanks to store the crafted fluids with fluid storage busses that have sticky cards. When i order a fluid the fluid gets made and sent to the tank properly but the recipe is not completed in the ae system.

**Comment by GlodBlock** [comment]
you must make the crafted fluid go through ae system. directly sending the fluid to the tank where the storage bus hooked on is invalid.

**Comment by mertgoksel** [comment]
I do not agree. Storage bus is a storage space that the system sees and can be accessed by the interfaces. Since its no different than a drive it should be valid. The fluid already goes through the system into the tank via fluid interface. Are you saying that the autocrafting only accounts for storage cells?

**Comment by GlodBlock** [comment]
hmm, then that should work. 
do you use a subnet to store stuffs? can you show the screenshot of your setup?

**Comment by mertgoksel** [comment]
![image](https://github.com/AE2-UEL/AE2FluidCraft-Rework/assets/52979890/b8c8107a-fdca-43ed-a85a-c0152b365015)
![image](https://github.com/AE2-UEL/AE2FluidCraft-Rework/assets/52979890/dfdc858c-0a6d-407b-bbe7-a6ece6e4c891)
![image](https://github.com/AE2-UEL/AE2FluidCraft-Rework/assets/52979890/cad6c8ab-1f9e-4a1f-8d3c-f75eca6e687c)

**Comment by mertgoksel** [comment]
I partitioned 3 fluid drives to have single fluids to solve the issue. But it doesn't change the fact that these should work. Also, all of this happens on main net.

**Comment by StaffiX** [comment]
Crafting CPUs store the items/fluids they're using.

Sticky card forces the items/fluids to only go to a specific storage, so it skips the cpu and goes to the supertank directly.

My recommendation would be to never use sticky cards on fluids that you autocraft.

**Comment by zeng-github01** [comment]
@GlodBlock You can close this issue.
